examples/topics/census.ipynb

```diff
@@ -615,23 +611,35 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "labels = gv.WMTS(\"http://tile.stamen.com/toner-labels/{Z}/{X}/{Y}.png\")\n",
     "population = dynspread(datashade(points, cmap=fire, element_type=gv.Image))\n",
-    "geomap * population * labels.options(level=\"annotation\")"
+    "gts.EsriImagery() * population * gts.StamenLabels().options(level=\"annotation\")"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "The dashboard.py example shows this same Census data in the context of an interactive dashboard, including color keys and hover information that help reveal the magnitudes at every location even while the plot faithfully reveals the structure."
+    "The dashboard.ipynb example shows this same census data in the context of an interactive dashboard, including color keys and hover information that help reveal the magnitudes at every location even while the plot faithfully reveals the structure."
    ]
   }
  ],
  "metadata": {
+  "kernelspec": {
+   "display_name": "Python 3",
+   "language": "python",
+   "name": "python3"
+  },
```

Comment:
Would be good to clear this metadata before merging this.  I use strip_kernel from Jean-Luc:

```
#!/bin/bash
cp "$1" /tmp/
/usr/local/bin/jq --indent 1 \
    '
    (.cells[] | select(has("outputs")) | .outputs) = []
    | (.cells[] | select(has("execution_count")) | .execution_count) = null
    | .metadata = {"language_info": {"name":"python", "pygments_lexer": "ipython3"}}
    | .cells[].metadata = {}
    ' "$1" > /tmp/stripped.ipynb
cat /tmp/stripped.ipynb > "$1"
```

Reply:
Thanks for the tip, will do.

Reply:
Done in 2f6b7ff